internal/cli/api/api.go

```diff
@@ -72,16 +75,41 @@ func convertAPIToCobraCommand(command api.Command) (*cobra.Command, error) {
 	shortDescription, longDescription := splitShortAndLongDescription(command.Description)
 
 	cmd := &cobra.Command{
-		Use:    commandName,
-		Short:  shortDescription,
-		Long:   longDescription,
-		Hidden: true, // TODO: part of CLOUDP-280743 Polish autogenerated docs
+		Use:   commandName,
+		Short: shortDescription,
+		Long:  longDescription,
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			// TODO: part of CLOUDP-280744 call execution framework
-			fmt.Println("Flags:")
-			cmd.Flags().Visit(func(f *pflag.Flag) {
-				fmt.Printf("%s: %s\n", f.Name, f.Value.String())
+			// TODO: wire up CLI and execution framework
+			// These lines are just here for testing purposes (only works for commands which don't need flags like `atlas api cluster listClusters`)
+			// Call "glue" layer which will glue commands and executor together: ticket CLOUDP-280745
+			// This layer will call something among the lines of:
+			executor, err := api.NewDefaultExecutor()
+			if err != nil {
+				return err
+			}
+
+			result, err := executor.ExecuteCommand(cmd.Context(), api.CommandRequest{
+				Command:    command,
+				Content:    cmd.InOrStdin(),
+				Format:     "json",                // get from flags
+				Parameters: map[string][]string{}, // get from flags + profile
+				Version:    "2023-01-01",          // get from flags
 			})
+
+			if err != nil {
+				return err
+			}
+
+			defer result.Output.Close()
+			_, err = io.Copy(os.Stdout, result.Output)
+			if err != nil {
+				return err
+			}
+
+			if !result.IsSuccess {
+				return errors.New("")
```

Comment:
what is this error?

Reply:
I'll improve this

Reply:
Added comments, this code was demo code. It'll be removed and replaced as part of CLOUDP-280745